frontend/device/android/device.lua

```diff
@@ -134,6 +134,9 @@ function Device:init()
             logger.dbg("Android application event", ev.code)
             if ev.code == C.APP_CMD_SAVE_STATE then
                 return "SaveState"
+            elseif ev.code == C.APP_CMD_DESTROY then
+                self:exit()
+                os.exit()
```

Comment:
Would a UIManager:quit or whatever the Exit menu does make sense here?

Reply:
Yeah, makes all the sense. It is also slightly more verbose :)